golden_data.Rmd

```diff
@@ -1,25 +1,51 @@
+Golden data in the context of biological sequence analysis are those that come from well-curated open-source databases. These data have been used to create all TreeSAPP reference packages to date, and they what should be used to create all future reference packages. But what does 'well-curated' mean, and how can you identify data that would meet this gold standard?
+
+## The golden standard
+
+The field of genomics is blessed with an abundance of reference data. In fact, it is not uncommon for biologists and bioinformaticians alike to gripe over their long-running computational tasks due to such large reference databases that need to be compared to. Not all of these data are, however, of the same quality. The vast majority have been automatically curated using algorithms with known shortcomings and only a small portion have been manually curated with verified functional characterization. These latter data are the ones we consider well-curated, or golden.
 
 ## What databases can I retrieve sequences from?
 
-Throughout the development of TreeSAPP I have frequented [FunGene](http://fungene.cme.msu.edu/), the functional gene pipeline and repository. This database provides curated sequences (of varying reliability) for many popular function and taxonomic anchor genes. FunGene is also great since they have hidden Markov models (HMMs) for every gene available, providing a sense of what the full length gene is and allowing users to filter the sequences based on the percentage of an HMM covered. Since we're building a reference tree I suggest downloading all full length and near-full length genes, so setting that HMM coverage parameter to between 60 and 90%.
+One of the most comprehensive and reputable biological sequence databases is SwissProt. The protein sequences curated within SwissProt can be accessed and downloaded through the [UniProt Knowledgebase](https://www.uniprot.org/) by selecting 'Swiss-Prot' under the 'UniProtKB' panel on the left of the landing page. You can submit queries for a protein family through their search bar, and add filters to the query using their [advanced search](https://www.uniprot.org/help/advanced_search) options. For example, you can search for reviewed RuBisCO sequences by entering the query 'rubisco taxonomy:"Bacteria [2]" AND reviewed:yes'.
+
+Throughout the development of TreeSAPP I have frequented [FunGene](http://fungene.cme.msu.edu/), the functional gene pipeline and repository. This database provides curated sequences (of varying reliability) for many popular functional and taxonomic anchor genes. FunGene is also great since they have hidden Markov models (HMMs) for every gene available, providing a sense of what the full length gene is and allowing users to filter the sequences based on the percentage of an HMM covered. Since we're building a reference tree I suggest downloading all full length and near-full length genes, so setting that HMM coverage parameter to between 60 and 90% is recommended.
```

Comment:
Can you explain what HMMs help with? I do not know so myself. I would focus on WHAT it does or WHY we use them (and not how).  Is HMM a prediction of how much of the gene in question is covered?